extensions/ql-vscode/src/query-history.ts

```diff
@@ -830,11 +833,20 @@ export class QueryHistoryManager extends DisposableObject {
     // In the future, we may support cancelling remote queries, but this is not a short term plan.
     const { finalSingleItem, finalMultiSelect } = this.determineSelection(singleItem, multiSelect);
 
-    (finalMultiSelect || [finalSingleItem]).forEach((item) => {
-      if (item.status === QueryStatus.InProgress && item.t === 'local') {
-        item.cancel();
+    const selected = finalMultiSelect || [finalSingleItem];
+    const results = selected.map(async item => {
+      if (item.status === QueryStatus.InProgress) {
+        if (item.t === 'local') {
+          item.cancel();
+        } else if (item.t === 'remote') {
+          void showAndLogInformationMessage('Cancelling remote query. This may take a while.');
```

Comment:
Probably?
```suggestion
          void showAndLogInformationMessage('Cancelling variant analysis. This may take a while.');
```

Reply:
Yes. There are some other places where we use "remote query" or "remote queries" in log messages that we should fix. I'll make the change here. And we should create a new PR for other instances of the old terminology.

Reply:
https://github.com/github/code-scanning/issues/5782